command-acceptor/app/controller/interfaces/CommandFactory.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+from app.models.ErrorMessage import ErrorMessage
+from app.models.Command import Command
+
+
+class CommandFactory(ABC):
+    @abstractmethod
+    def command(self) -> Command:
+        raise NotImplementedError()
+
+    @abstractmethod
+    def error_message(self) -> ErrorMessage:
```

Comment:
Why is this interface declaring two different operations?
You name interface ```..Factory``` factory has to construct something and this something should be one Entity only.

Reply:
Done. But not sure about implementation. I am returning message as a result of process method. Maybe message should be available by .message method